dist/pf-relative.js

```diff
@@ -90,13 +98,28 @@ var noop = function noop () {};
 
 module.exports = function RelativeDate (date) {
   if (!(this instanceof RelativeDate)) return new RelativeDate(date);
-  date = date ? new Date(date) : new Date();
 
+  var lastTimeout = null;
```

Comment:
Maybe `lastTick` is a better name? Given we have a `clearTick` function down below...